concepticondata/conceptlists/Scheible-2014-1755a.tsv

```diff
@@ -0,0 +1,1756 @@
+Scheible-2014-1755a-194	194	Funk	wireless	noun			1704:Funk-Kabel:0 1708:Funk-Rauchzeichen:0 1892:Funk-Feuerwehr:0 1896:Funk-Kommunikation:0 2078:Funk-Radio:0 2082:Funk-Radiowellen:0 2404:Funk-Feuerwehr:0 2405:Funk-Kabel:0 2406:Funk-Kommunikation:0 2407:Funk-Radio:0 2408:Funk-Radiowellen:0 2409:Funk-Rauchzeichen:0 2852:Funk-Kabel:0 2856:Funk-Rauchzeichen:0 3040:Funk-Feuerwehr:0 3044:Funk-Kommunikation:0 3226:Funk-Radio:0 3230:Funk-Radiowellen:0	→282 →439 →176 →302 →436 →437 →176 →282 →302 →436 →437 →439 →282 →439 →176 →302 →436 →437	ANT ANT ANT ANT ANT ANT HYP HYP HYP HYP HYP HYP SYN SYN SYN SYN SYN SYN	5.5 4.3333 2.3333 1 0.6667 1.3333 1.3333 1.3333 6.5 3.3333 5.5 0 0.8333 1.3333 0.6667 4 5.3333 5
+Scheible-2014-1755a-195	195	Fuß	foot	noun	1301	FOOT	1947:Bein-Fuß:1 2298:Bein-Fuß:1 3095:Bein-Fuß:1	←71 ←71 ←71	ANT HYP SYN	2.3333 1.6667 2.1667
+Scheible-2014-1755a-196	196	Fußboden	floor	noun	1632	FLOOR	1806:Wandzeitung-Fußboden:1 2765:Wandzeitung-Fußboden:1 2954:Wandzeitung-Fußboden:1	←594 ←594 ←594	ANT HYP SYN	2.6667 0 0.1667
+Scheible-2014-1755a-197	197	Fügung	coincidence	noun			1672:Explosion-Fügung:1 2383:Explosion-Fügung:1 2820:Explosion-Fügung:1	←167 ←167 ←167	ANT HYP SYN	1.6667 0 0.1667
+Scheible-2014-1755a-198	198	Führer	leader	noun			2130:Großmeister-Führer:1 2446:Großmeister-Führer:1 3278:Großmeister-Führer:1	←246 ←246 ←246	ANT HYP SYN	0.8333 2.6667 5.5
+Scheible-2014-1755a-199	199	Fürargument	in favor of your reasoning (as in "pros and cons")	noun			1724:Gegenargument-Fürargument:1 2418:Gegenargument-Fürargument:1 2872:Gegenargument-Fürargument:1	←209 ←209 ←209	ANT HYP SYN	7.3333 0.6667 1.6667
+Scheible-2014-1755a-200	200	Gast	guest	noun	1782	GUEST	1747:Bedienung-Gast:1 2293:Bedienung-Gast:1 2895:Bedienung-Gast:1	←67 ←67 ←67	ANT HYP SYN	6.3333 0.3333 0.3333
+Scheible-2014-1755a-201	201	Gatter	gate	noun			1712:Gatter-Einfahrt:0 1716:Gatter-Freiheit:0 1900:Gatter-Begrenzung:0 1904:Gatter-Zaun-Tor:0 2086:Gatter-Umzäunung:0 2090:Gatter-Zaun:0 2410:Gatter-Begrenzung:0 2411:Gatter-Einfahrt:0 2412:Gatter-Freiheit:0 2413:Gatter-Umzäunung:0 2414:Gatter-Zaun:0 2415:Gatter-Zaun-Tor:0 2860:Gatter-Einfahrt:0 2864:Gatter-Freiheit:0 3048:Gatter-Begrenzung:0 3052:Gatter-Zaun-Tor:0 3234:Gatter-Umzäunung:0 3238:Gatter-Zaun:0	→140 →190 →69 →613 →546 →612 →69 →140 →190 →546 →612 →613 →140 →190 →69 →613 →546 →612	ANT ANT ANT ANT ANT ANT HYP HYP HYP HYP HYP HYP SYN SYN SYN SYN SYN SYN	1.3333 3.6667 0.6667 0.8333 0.3333 1.1667 6.8333 2.3333 0 6.6667 5.1667 2.6667 3.1667 0.5 4.8333 6.5 6.8333 6
```

Comment:
Would GATE 2940 be appropriate here?

Reply:
"I'd unmap as it's a type of gate only" comment by @Schweikhard regarding this mapping.